src/utils/handleRequestWith.ts

```diff
@@ -89,6 +95,9 @@ export const handleRequestWith = (
           response,
         )
 
+        context.emitter.emit('response:bypass', req)
```

Comment:
Would it make sense to add the response here, if that's possible?
Because for the mocked instances, we do add the response to the emitter.

Reply:
Yes, this is one of my todo items. Once I refactor the worker-client signaling system, the library will receive a `RESPONSE_ORIGINAL` (or similar) event from the worker when an original response is retrieved. This `response:bypass` event should be executed at that time, not during request handling.

Reply:
Request end and response retrieval happen in two different moments of time and two different contexts. Therefore, they will be handled separately, with you being able to connect them post-factum by the persistent `req.id` and `response.requestId`.